core/core/test/serverspec/spec/zookeeper/zookeeper_spec.rb

```diff
@@ -1,27 +1,81 @@
 require 'spec_helper'
 
-describe service('zookeeper') do
-  it { should be_enabled }
-  it { should be_running }
+describe 'Checking if zookeeper service is running' do
+  describe service('zookeeper') do
+    it { should be_enabled }
+    it { should be_running }
+  end
 end
 
-describe port(2181) do
-  it { should be_listening }
-end
+describe 'Checking if the ports are open' do
+  describe port(2181) do            #2181 for client connections
+    let(:disable_sudo) { false }  
+    it { should be_listening }
+  end
+
+  # These ports are not always open. Depends on configuration.
```

Comment:
Commented out code is rather bad practice, maybe it would be better to remove it or add a condition whether to run it or skip.

Reply:
👍